tests/s390x_tests/consoletest_TOOL_s390_vmcp.pm

```diff
@@ -23,12 +21,7 @@ sub run {
     my $self = shift;
     $self->copy_testsuite('TOOL_s390_vmcp');
     $self->execute_script('vmcp_main.sh');
-    $self->cleanup_testsuite('vmcp');
-}
-
-sub post_fail_hook {
-    my $self = shift;
-    $self->export_logs();
+    $self->cleanup_testsuite('TOOL_s390_vmcp');
```

Comment:
if you declare a post_run_hook in s390base, you could reduce the need for the cleanup, and with a pre_run the need for copying it, just making sure that there's a test variable that prepares everything. - Maybe let's discuss it offline, and come with improvements for this, if they actually prove to be useful :P)


But I'm sure that the name of the test module can be obtained in an easier way
```suggestion

sub    pre_run_hook {
       $self->copy_testsuite(s/consoletest_(.*)\.pm/$1/ __FILE__);
}

sub    post_run_hook {
        $self->cleanup_testsuite(s/consoletest_(.*)\.pm/$1/ __FILE__);
}
```

Reply:
ah, I forgot such hooks exist.

Reply:
Actually, yes, the name of the test module is on the settings.